phoebe/atmospheres/passbands.py

```diff
@@ -1142,6 +1121,7 @@ def compute_intensities(self, atm, path, impute=False, include_extinction=False,
         # self._ck2004_boosting_photon_grid = np.nan*np.ones((len(self.atm_axes['ck2004'][0]), len(self.atm_axes['ck2004'][1]), len(self.atm_axes['ck2004'][2]), len(self.atm_axes['ck2004'][3]), 1))
 
         if include_extinction:
+            # ? Should this not include mus as well?
```

Comment:
should it?

Reply:
I think it should, but this can also be deferred. Here's the [action item](https://github.com/orgs/phoebe-project/projects/2/views/11?pane=issue&itemId=57136993).

Reply:
Originally we decided to just derive the extinction corrections based on Inorms so as to not massively increased the passband file sizes.  Given the magnitude of the correction from extinction, I still think this is fine (and is the way we say that we do it in the paper).  Limb-darkening changes the SED but also reduces the brightness, so the majority of the flux comes from the centre where the SED is very, very similar to Inorm. Changing to Imu would would increase passband sizes by at least an order of magnitude for negligible gain.